cni/network/invoker_azure_test.go

```diff
@@ -272,7 +274,29 @@ func TestAzureIPAMInvoker_Delete(t *testing.T) {
 				nwInfo: getNwInfo("10.0.0.0/24", "2001:db8:abcd:0012::0/64"),
 			},
 			args: args{
-				address: getCIDRNotationForAddress("2001:db8:abcd:0015::0/64"),
+				addresses: []*net.IPNet{
+					getCIDRNotationForAddress("2001:db8:abcd:0015::0/64"),
+				},
+				nwCfg: &cni.NetworkConfig{
+					IPAM: cni.IPAM{
+						Address: "2001:db8:abcd:0015::0/64",
+					},
+				},
+			},
+		},
+		{
+			name: "delete happy path ipv4+ipv6",
```

Comment:
since the delete() signature is not changed, so no case to for deleting both ipv4 and ipv6 addresses. Failed test cases are there:
"error on v6 delete"
"error on v4 delete"
"error address is nil"